lib/Models/CatalogItem.js

```diff
@@ -86,6 +87,13 @@ var CatalogItem = function(terria) {
      * @type {String}
      */
     this.dataCustodian = undefined;
+    /**
+     * Gets or sets an attribution text of this data item.
+     * This property is an HTML string that must be sanitized before display to the user.
+     * This property is observable.
+     * @type {String}
+     */
+    this.textAttribution = undefined;
```

Comment:
Should this just be `attribution`?  Currently it can be both text and a link.  In the future we might add support for an image, and we wouldn't want to change the name of this top-level property to do so.